app/controllers/api/v1/api_controller.rb

```diff
@@ -15,7 +15,8 @@ class ApiController < ApplicationController
       # Raises UnauthorizedRequestError exception on invalid token lookup
       # Returns a Tenant
       def authenticate!
-        Tenant.find_by!(api_key: token)
+        tenant = Tenant.find_by!(api_key: token)
+        tenant.update(request_count: tenant.request_count + 1)
```

Comment:
Considered moving this into the `Tenant` model, but decided not to since this seemed like a fairly basic operation.